### PR TITLE
fix: tests passing and configuring Travis CI

### PR DESCRIPTION
**Pull request by obiyankenobi**

### CODE REVIEW

File: __tests__/date.test.js [hunk 1]
```diff
@@ -7,12 +7,11 @@
 
 import dateFormatter from '../src/date';
 
-
 test('Parse timestamp', () => {
   // 2019-05-08 11:43:49
   const timestamp = 1557312229;
-  const formatted = dateFormatter.parseTimestamp(timestamp);
-  expect(formatted).toBe('5/8/2019 11:43:49 AM');
+  const formatted = dateFormatter.parseTimestamp(timestamp, 'UTC');
```
Comment (msbrogli) [inline __tests__/date.test.js]: As it returns a string, shouldn't we call it `timestampToString` or something like it?

Comment (obiyankenobi) [reply __tests__/date.test.js]: I believe this is not related to this PR.

This would also create additional tasks on wallet desktop and wallet lib.

Comment (msbrogli) [reply __tests__/date.test.js]: Ok. I think we should open an issue related to it, and we can skip it in this PR.

File: __tests__/date.test.js [hunk 2]
```diff
@@ -7,12 +7,11 @@
 
 import dateFormatter from '../src/date';
 
-
 test('Parse timestamp', () => {
   // 2019-05-08 11:43:49
```
[the remark this answers is not in the document]
Comment (obiyankenobi) [reply __tests__/date.test.js]: Done

**Pull request merged**